<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Penunjung Perpustakaan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="misc.qrc">
    <normaloff>:/unsoed.png</normaloff>:/unsoed.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {
	
	background-color: #00adb5;
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <property name="styleSheet">
      <string notr="true">QWidget#loginPage {
	background-color: #222831;	
	border-radius: 10px;
}</string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>421</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox#groupBox {
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
	font-weight: 400;
}</string>
      </property>
      <property name="title">
       <string>LOGIN PENGUNJUNG</string>
      </property>
      <widget class="QLineEdit" name="noIDEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#noIDEdit {
	border-radius: 5px;
	padding-left: 10px;
	font: 14pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>Masukkan No ID / Nama</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#loginButton {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}
QPushButton#loginButton:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 16pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="loginGuestButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#loginGuestButton {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	color: #eeeeee;
	font: 14pt &quot;Comic Sans MS&quot;;
}
QPushButton#loginGuestButton:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 16pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="text">
        <string>LOGIN TAMU</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="loginGuest">
     <property name="styleSheet">
      <string notr="true">QWidget#loginGuest {
	background-color: #222831;	
	border-radius: 10px;
}</string>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>431</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox#groupBox_3 {
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
	font-weight: 400;
}</string>
      </property>
      <property name="title">
       <string>LOGIN TAMU</string>
      </property>
      <widget class="QLineEdit" name="namaTamuEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#namaTamuEdit {
	border-radius: 5px;
	padding-left: 10px;
	font: 14pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>Masukkan Nama</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="instansiTamuEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#instansiTamuEdit {
	border-radius: 5px;
	padding-left: 10px;
	font: 14pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Masukkan Instansi</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginTamuButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#loginTamuButton {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}
QPushButton#loginTamuButton:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 16pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="kembaliButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#kembaliButton_2 {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}
QPushButton#kembaliButton_2:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 16pt &quot;Comic Sans MS&quot;;
}</string>
       </property>
       <property name="text">
        <string>KEMBALI</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="userPage">
     <property name="styleSheet">
      <string notr="true">QWidget#userPage {
	background-color: #222831;	
	border-radius: 10px;
}</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#label {
	font-weight: 500;
	font: 18pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}</string>
       </property>
       <property name="text">
        <string>IDENTITAS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>361</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label1">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#label1 {
	font: 11pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string>No ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="noIDLabel">
            <property name="styleSheet">
             <string notr="true">QLabel#noIDLabel{
	font: 12pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#label2{
	font: 11pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string>Nama</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="namaLabel">
            <property name="styleSheet">
             <string notr="true">QLabel#namaLabel {
	font: 12pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label3">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#label3 {
	font: 11pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string>Alamat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="alamatLabel">
            <property name="styleSheet">
             <string notr="true">QLabel#alamatLabel {
	font: 12pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
	color: #eeeeee;
}</string>
            </property>
            <property name="lineWidth">
             <number>6</number>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label4">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#label4 {
	font: 11pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="styleSheet">
             <string notr="true">QLabel#statusLabel {
	font: 12pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
	color: #eeeeee;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>361</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="kembaliButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#kembaliButton {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}
QPushButton#kembaliButton:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 16pt &quot;Comic Sans MS&quot;;
}</string>
          </property>
          <property name="text">
           <string>KEMBALI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lanjutLoginButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#lanjutLoginButton {
	border-radius: 5px;
	background-color: rgba(0, 173, 181, 220);
	font: 14pt &quot;Comic Sans MS&quot;;
	color: #eeeeee;
}
QPushButton#lanjutLoginButton:hover {
	background-color: rgba(0, 173, 181, 255);
	font: 15pt &quot;Comic Sans MS&quot;;
}</string>
          </property>
          <property name="text">
           <string>LANJUT LOGIN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>noIDEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>loginGuestButton</tabstop>
  <tabstop>kembaliButton</tabstop>
  <tabstop>lanjutLoginButton</tabstop>
 </tabstops>
 <resources>
  <include location="misc.qrc"/>
 </resources>
 <connections/>
</ui>
